### PR TITLE
Add Privacy Manifest

### LINKED ISSUE #319: Apple's update to their API policy - Required Reason in Privacy manifest
Hi,

From Fall 2023 Apple starts rolling out new privacy requirements, The apps and third-party SDKs should contain a privacy manifest file when they access required reason API. TrustKit SDK uses [NSUserDefaults].(https://developer.apple.com/documentation/foundation/nsuserdefaults) which is required description of use in PrivacyManifest.
Otherwise, apps that don’t describe their use of required reason API in their privacy manifest file won’t be accepted by App Store Connect. Here is a link to the [Apple documentation].(https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api).

Do you have plans to add privacy manifest files to the TrustKit?

### PR DESCRIPTION
**Pull request by uroboro**
Fixes #319

### CODE REVIEW

File: TrustKit/PrivacyInfo.xcprivacy [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```
Comment (manahilov) [inline TrustKit/PrivacyInfo.xcprivacy:4]: For Manifest completeness you can include the tracking KVPs as well.
```
    <key>NSPrivacyTracking</key>
    <false/>
    <key>NSPrivacyCollectedDataTypes</key>
    <array/>
    <key>NSPrivacyTrackingDomains</key>
    <array/>
```

**Review by pedrohls94 — APPROVED** [review]

**Review by EthanArbuckle — APPROVED** [review]

**Comment by iOSNinja** [comment]
Hello,
When will this change be available via the version update?

Thanks.

**Comment by uroboro** [comment]
3.0.4 has been released for SPM and CocoaPods

**Comment by mccarron** [comment]
@uroboro Can you also mark it as a release, some scripts won't pull if it's only a tag.

**Pull request merged**